feat(dev-portal): Key auth expiration (#4009)
Kong/developer.konghq.com: app/_how-tos/enable-key-auth-for-dev-portal.md

faqs:
  - q: My published API is private in Dev Portal, how do I allow developers to see it?
    a: If an API is published as private, you must enable Dev Portal RBAC and [developers must sign in](/dev-portal/developer-signup/) to see APIs.
  - q: Can I configure they key auth key to expire?
    a: |

>         Yes. To configure the key to expire at a certain time, do the following:

Reviewer: ```suggestion
        Yes. To configure expiration for a key, do the following:
```
